chart/values.yaml

```
@@ -293,6 +293,8 @@ csi:
     tolerations: []
     # -- Set PriorityClass, overrides global
     priorityClassName: ""
+    # -- Set `--prevent-volume-mode-conversion` in csi-snapshot-controller and csi-provisioner to prevent modifying the volume mode when creating a PVC from an existing VolumeSnapshot
```

How we do it is internal impl

```suggestion
    # -- Prevent modifying the volume mode when creating a PVC from an existing VolumeSnapshot
```